app/controllers/subscription_controller.rb

```diff
@@ -117,57 +117,58 @@ def digest
   end
 
   def multiple_add
-    if !params[:tagnames] || params[:tagnames] == ''
+    return_to = params[:return_to] || "/subscriptions?_=" + Time.now.to_i.to_s
+    if params[:tagnames].blank?
       flash[:notice] = "Please enter tags for subscription in the url."
-      redirect_to "/subscriptions" + "?_=" + Time.now.to_i.to_s
```

Comment:
Can you please see why git is showing many lines of change eventhough it is two line change?

Reply:
sorry 4 lines of change